database/service_db_test.go

```diff
@@ -48,6 +48,21 @@ func TestServiceDBSave(t *testing.T) {
 	require.EqualError(t, db.Save(s), errAliasSameLen.Error())
 }
 
+func TestServiceDBSaveWithSameAlias(t *testing.T) {
+	db, closer := openServiceDB(t)
+	defer closer()
+
+	s := &service.Service{ID: "01", Alias: "a", Name: "test-service"}
+	require.NoError(t, db.Save(s))
+	s = &service.Service{ID: "02", Alias: "a", Name: "test-service"}
+	require.Equal(t, &ErrSameAlias{"a"}, db.Save(s))
+
+	_, err := db.Get("1")
```

Comment:
Why to get `"1"` and `"2"` (as they are unrelated to this test), probably you wanted `"01"` and `"02"`